### PR TITLE
Api page update

### PR DESCRIPTION
**Pull request by dennisnewel**
Updating the content a bit and changing the section about API v1 to mention deprecation

should hold off on deploying this until API release date

### CODE REVIEW

**Comment by mlocher** [comment]
@dennisnewel the error on the build is because another page links to this page. Those links make use of a Jekyll helper named [`link`](https://jekyllrb.com/docs/templates/#link) that works by passing in the filename of the object you want to link to.

Because you moved the file (in addition to publishing it at a new URL), we also need to update all calls to `link` that point to this file. I'd search the repository for mentions of `_basic/builds-and-configuration/api.md` (taken from the error message) and then update those references to the new location.

```
Liquid Exception: Could not find document '_basic/builds-and-configuration/api.md' in tag 'link'. Make sure the document exists and the path is correct. in /docs/_general/projects/getting-started.md
jekyll 3.6.2 | Error:  Could not find document '_basic/builds-and-configuration/api.md' in tag 'link'.
```

File: _general/integrations/api.md [hunk 1]
```diff
@@ -24,14 +25,14 @@ We recently launched a new version of the API as a public Beta, which is open to
 
 If you're just looking to get started, head over to the [API v2 Documentation](https://apidocs.codeship.com/v2/) for all the details.
 
-## Overview
+## Core Use Cases
 
-The new API have initially been focused on solving a specific set of use cases, that have been requested by the majority of users:
+The new API have initially been focused on solving a specific set of core use cases, which have been requested by the majority of users:
```
Comment (ethangj) [inline _general/integrations/api.md]: has initially, has been requested

File: _general/integrations/api.md [hunk 2]
```diff
@@ -24,14 +25,14 @@ We recently launched a new version of the API as a public Beta, which is open to
...
+* Chaining Projects - what we've heard most here is needing to trigger a downstream project based on results from one or more upstream projects, and potentially information from outside of codeship. If you're looking to chain multiple Codeship projects, consider relying on a notification webhook as the trigger instead of polling the API for updates to the upstream build.
+* Automatically (or manually) Trigger New Builds - scheduled builds, builds triggered by events in external systems, or just giving QA a button on a webpage to trigger a build, have been among the requests for this use case.
+* Automatic project creation - for those who more frequently create new projects, there's been a lot of requests for being able to automatically provisioning a project on codeship, together with provisioning a new repo, test environment etc. to make bootstrapping a new project quicker and more standardized
```
Comment (ethangj) [inline _general/integrations/api.md]: what do you think of some support CTA? if you have something you're trying to accomplish or get working with the API, please let us know (email helpdesk@)

Comment (dennisnewel) [reply _general/integrations/api.md]: that's a good idea. i'll add that in

**Pull request merged**